config/kmesh_marcos_def.h

```diff
@@ -14,15 +14,15 @@
  * and sidecar containers. The bpf_sk_original_addr auxiliary function is
  * used to obtain the original destination address.
  */
-#define MDA_NAT_ACCEL 1
+#define MDA_NAT_ACCEL 0
```

Comment:
please revert,  you need to run make clean after building.

Reply:
my fault